### ISSUE TITLE
Command List Empty in 0.4.0

### ISSUE THREAD

**Issue by iamrecursion** [AlexBieg/obsidian-better-command-palette#14]
It seems like there's a missing null check for starred command items. If you don't have any (or have the stars plugin disabled), it crashes when opening the command palette.

```
Command failed to execute:  obsidian-better-command-palette:open-better-commmand-palette
VM162:320 TypeError: Cannot read property 'map' of undefined
    at m.getPinnedItems (eval at <anonymous> (app.js:1), <anonymous>:15:2937)
    at m.getSortedItems (eval at <anonymous> (app.js:1), <anonymous>:15:2574)
    at m.getItems (eval at <anonymous> (app.js:1), <anonymous>:15:3223)
    at m.getSuggestionsAsync (eval at <anonymous> (app.js:1), <anonymous>:15:3538)
    at m.getSuggestions (eval at <anonymous> (app.js:1), <anonymous>:15:3894)
    at m.e.updateSuggestions (app.js:1)
    at m.e.onOpen (app.js:1)
    at m.t.open (app.js:1)
    at Object.callback (eval at <anonymous> (app.js:1), <anonymous>:15:16404)
    at SR (app.js:1)
```

**Comment by AlexBieg** [comment]
Oops. I should have checked that. I'll get a fix out for that soon. I am currently working on refactoring the code to make development easier because I was initially lazy with planning 😄 I'll roll this fix into that update.

**Comment by iamrecursion** [comment]
Oopsie daisy! Looking forward to the fix!

**Comment by AlexBieg** [comment]
Just released `0.5.0`. It should fix this bug for you in addition to some other nice to have. Gonna close this for now but feel free to reopen if you are still having issues.

**Comment by iamrecursion** [comment]
This has fixed it! Thank you so much for such a quick response on this and the performance issue! This plugin has made such a vital difference to how I use Obsidian!
